Dockerfile

```diff
@@ -1,39 +1,24 @@
-FROM openbuildservice/base
-ARG CONTAINER_USERID
-
-# FIXME: https://bugzilla.opensuse.org/show_bug.cgi?id=957818
-RUN rm -rf /var/cache/zypp/*
+# This is just a thin layer on top of the frontend container
+# that makes sure different users can run it without the
+# contained rails app generating files in the git checkout with
+# some strange user...
 
-# Install requirements for the frontend 
-RUN zypper -n install --no-recommends --replacefiles \
-  # as search daemon
-  sphinx \
-  # for testing javascript driven pages
-  phantomjs \
-  nodejs \
-  # for accessing the database
-  mariadb-client \
-  # for rspec (it does something with git)
-  git-core \
-  # dependencies for building gems
-  ruby2.4-devel cyrus-sasl-devel openldap2-devel libxml2-devel zlib-devel libxslt-devel \
-  # gems we don't want to build
-  ruby2.4-rubygem-mysql2 \
-  # gems we use that are not in our bundle
-  ruby2.4-rubygem-bundler ruby2.4-rubygem-thor-0_19 ruby2.4-rubygem-foreman
+FROM openbuildservice/frontend-base
+ARG CONTAINER_USERID
 
 # Configure our user
 RUN usermod -u $CONTAINER_USERID frontend
 
+ADD src/api/Gemfile /obs/src/api/Gemfile
```

Comment:
@hennevogel, why is this code duplicated in [Dockerfile.frontend-base](https://github.com/openSUSE/open-build-service/pull/3900/commits/f9ea92b52bcec451725255ac5735ec87953f2362#diff-0b2aec5dc9d5d0c9fd05f8ff48cf8467R27)?

Reply:
This serves as a *cache* of some sort. Otherwise every build in travis would build the 'frontend' container (as it's only available when you build) and hence would fetch all gems again...

I imagine the cache to work like this

1. We build the *frontend-base* container and upload it to the docker hub with the gems from that time
  1.1 Someone triggers a travis build with a commit that leaves the bundle untouched
  1.2 Someone triggers a travis build with a commit that changes the bundle
3. In case of 1.1 nothing happens
4. In case of 1.2 our master build will rebuild and publish the *frontend-base* container
5. Repeat

Reply:
@hennevogel, thanks for the explanation :smile_cat: